### DISCUSSION TITLE
Realtime update active timer -> task

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [schmidfelix/timey#2] [Ideas] [answered]
It would be awesome if the active timer could auto update the time / costs

**Comment by schmidfelix** [comment, marked as answer]
That would be nice indeed, I'm going to take a look at that, but I'll have to make sure that the performance doesn't suffer. I'll keep you updated 😉

**Reply by SebastiaanKloos** [reply]
Thanks! No need to rush!😎
